### PR TITLE
Add episode_num helper and tests

### PR DESCRIPTION
**Pull request by labrys**
Usage:

``` python
from sickrage.helpers.common import episode_num

print('Title - %s - Episode Title' % episode_num(season, episode))
print('Title - %s - Episode Title' % episode_num(episode, numbering='absolute'))
```

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: sickrage/helper/common.py [hunk 1]
```diff
@@ -302,3 +302,24 @@ def try_int(candidate, default_value=0):
...
+    :returns: a string in s01e01 format or absolute numbering
+    """
+
+    numbering = kwargs.pop('numbering', 'standard')
+
+    if numbering == 'standard':
+        if season is not None and episode:
+            return 'S{0:0>2}E{1:02}'.format(season, episode)
+    elif numbering == 'absolute':
+        if not (season and episode) and (season or episode):
```
Comment (fernandog) [inline sickrage/helper/common.py:323]: In the future we should check:

scene_season If scene_season else season
scene_episode If scene_episode else episode
scene_absolube_num if scene_absolute_num else  absolute_number

**Comment by labrys** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/SickRage/SickRage/pull-request/782/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=SickRage/SickRage)

File: sickrage/helper/common.py (hunk 1 above)
Comment (labrys) [inline sickrage/helper/common.py:323]: lol was just saying that in irc and that we can update numbering to take a list so it tries in the order specified.

**Pull request merged**